kinto/core/__init__.py

```diff
@@ -51,6 +51,7 @@
         'kinto.core.initialization.setup_storage',
         'kinto.core.initialization.setup_permission',
         'kinto.core.initialization.setup_cache',
+        'kinto.core.initialization.setup_hawk_auth',
```

Comment:
I believe instead of putting it in core we should create a hawk plugin in the plugins repository.

Reply:
~~repository~~ → folder :)